src/conversion_helper.js

```diff
@@ -903,22 +965,41 @@ function formatConversion(conversions) {
     const roundedConversions = conversion['rounded'];
     if (Array.isArray(roundedConversions)) {
       conversion['formatted'] = roundedConversions.map(rc => {
-        return createMap(rh.addCommas(rc['number']), rc['unit']);
+        let result = {
+          'numbers': [],
+          'unit': rc['unit']
+        };
+        rc['numbers'].forEach(function(item) {
+            result['numbers'].push(rh.addCommas(item));
+        });
+        return result;
       });
     } else {
-      const metric = conversion['rounded']['number'];
-      conversion['formatted'] = createMap(rh.addCommas(metric), conversion['rounded']['unit'])
+      const metric = conversion['rounded']['numbers'];
+      let result = {
+        'numbers': [],
+        'unit': conversion['rounded']['unit']
+      };
+      metric.forEach(function(item) {
+          result['numbers'].push(rh.addCommas(item));
+      });
+      conversion['formatted'] = result;
     }
 
     const imperialUnit = conversion['imperial']['unit'];
-    const imperialNumber = conversion['imperial']['number'];
+    const imperialNumbers = conversion['imperial']['numbers'];
 
     const postprocessInput = unitLookupMap[imperialUnit]['postprocessInput'];
+    conversion['imperial']['numbers'] = [];
     if (postprocessInput) {
-      conversion['imperial']['number'] = postprocessInput(imperialNumber);
+      imperialNumbers.forEach(function(item) {
+        conversion['imperial']['numbers'].push(postprocessInput(item));
+      });
```

Comment:
I think the code would be clearer if we wrote `conversion['imperial']['numbers'] = imperialNumbers.map(postprocessInput)`